api/internal/handler/data_loader/loader/openapi3/openapi3.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package openapi3
+
+import (
+	"regexp"
+
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+type OpenAPISpecFileType string
+
+type Loader struct {
+	// MergeMethod indicates whether to merge routes when multiple HTTP methods are on the same path
+	MergeMethod bool
+	// TaskName indicates the name of current import/export task
+	TaskName string
```

Comment:
How to decide on this field? Two scenarios should be confirmed:

1. Convert the same OAS file twice, with different `TaskName`, then duplicated routes will be created but their functionalities are the same.
2. Convert the two OAS files with the same `TaskName`, and the path and method are the same, will the routes conflict with each other?

Reply:
@tokers 

For this module, it is not concerned with duplicate routes, but only with the parsing and conversion of OAS documents, and for the parts of the duplicate routes that you pointed out as limitations, you can see them here #2474 : https://github.com/apache/apisix-dashboard/blob/7fb05943de573d782b42c18728e4dc9990e04cf3/api/internal/handler/data_loader/route_import.go#L149-L197

Reply:
OK.